server/src/main/java/org/apache/cassandra/sidecar/tasks/PeriodicTaskExecutor.java

```diff
@@ -66,53 +71,125 @@ public PeriodicTaskExecutor(ExecutorPools executorPools, ClusterLease clusterLea
      * Schedules the {@code task} iff it has not been scheduled yet.
      *
      * @param task the task to execute
-     * @return the identifier of the timer associated with this task
      */
-    public long schedule(PeriodicTask task)
+    public void schedule(PeriodicTask task)
```

Comment:
Use `unschedule` method to unschedule task. It is improper to use timerID to unschedule, which cancels the need of exposing timerId.